docker-compose.yml

```
@@ -1,14 +1,15 @@
 version: '2'
 services:
   web:
-    image: fpfis/httpd-php-dev:7.3
+    image: fpfis/httpd-php:8.0-dev
```

Why we use 8.0-dev:
```suggestion
    image: fpfis/httpd-php-dev:8.0
```
or 8.1?

---

I got the available image from this ticket https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-1778
We'll stick to PHP8 as discussed.

---

Other components use `fpfis/httpd-php-dev:8.0`:
https://github.com/openeuropa/code-review/pull/142/files
https://github.com/openeuropa/task-runner/pull/160/files